src/components/DatePicker/DatePicker.jsx

```diff
@@ -97,52 +112,59 @@ export const DatePicker = ({
         onMonthChange?.(m);
     };
 
+    const handleStartMonthChange = (m) => {
+        setStartMonth(m);
+        onMonthChange?.(m);
+    };
+
+    const handleEndMonthChange = (m) => {
+        setEndMonth(m);
+        onMonthChange?.(m);
+    };
+
     const captionElement = shouldShowYearPicker
         ? ({ date }) => <MonthYearSelector date={date} currentMonth={currentMonth} onChange={handleMonthChange} />
         : undefined;
 
-    const isDisabled = (date) => {
+    const captionStartElement = shouldShowYearPicker
+        ? ({ date }) => <MonthYearSelector date={date} currentMonth={currentMonth} onChange={handleStartMonthChange} />
+        : undefined;
+
+    const captionEndElement = shouldShowYearPicker
+        ? ({ date }) => <MonthYearSelector date={date} currentMonth={currentMonth} onChange={handleEndMonthChange} />
+        : undefined;
+
+    const rangeModifier = isRangeVariant ? { start: value.from, end: value.to } : null;
+
+    const disabledStartDays = (date) => {
+        if (isFunction(disabledDays)) {
+            return disabledDays(date) || dayjs(date).isAfter(value.to);
```

Comment:
You'll have to check using the seller's timezone which is passed as a prop. See the `handleTodayClick` method logic. Make sure to test the change once you write it.